pinner/progress/progress.go

```diff
@@ -0,0 +1,3 @@
+package pinning_progress
```

Comment:
nit: `pinning_progress` -> `progress` for idiomatic package name and so it can be imported as `/pinner/progress`